src/event_loop.rs

```diff
@@ -343,7 +343,7 @@ impl<T> EventLoop<T>
 
             let poll_opts = PollOpt::edge() | PollOpt::oneshot();
 
-            let tokens = [1, 2];
+            let tokens = [1, 2, 3];
```

Comment:
Obviously we know that one token is shared for both read + write on unix, so we could use `2` for the signal pipe... but better safe than sorry, right?